### DISCUSSION TITLE
Enquiries Switch Turning off on its own

### DISCUSSION THREAD

**Discussion by mattpitts74** [mKeRix/room-assistant#581] [Q&A]
Hi There, 

I have just installed and setup room assistant on a couple of  PI's in my house for testing.  I have noticed that both the Enquiries switches for both of the Pi's keep turning off on their own?? 

Is this supposed to happen?  How do i stop it? as once-off, they do not pick up my phone again until I switch them back on manually.  Any help would be much appreciated.

**Comment by mKeRix** [comment]
It's not supposed to happen, there is no code in room-assistant that would cause the switch to toggle at any point. I'd suggest to check:

- if you set `inquireFromStart` (and if so: to what value?)
- if you have any automations in Home Assistant that would toggle switches
- if you have anything else that might write into the MQTT command topic for these switches
